components/x-article-save-button/readme.md

```diff
@@ -0,0 +1,49 @@
+# Article Save Button
+
+This module provides a template for myFT save article buttons, and is intended to replace and is intended to replace the legacy handlebars component in [n-myft-ui](https://github.com/Financial-Times/n-myft-ui/tree/master/components/save-for-later).
+
+## Installation
+
+```bash
+npm install --save @financial-times/x-article-save-button
+```
+
+## Props
```

Comment:
I think this needs a follow up "usage" section, might even be able to copy and paste from here!

https://github.com/Financial-Times/x-dash/tree/master/components/x-teaser#usage

Reply:
I thought this was sufficiently covered in a generic way [here](https://financial-times.github.io/x-dash/docs/integration/using-components#rendering), but I'm happy to add a specific section in here too.